src/blr/blr.yaml

```diff
@@ -4,7 +4,8 @@ molecule_tag: MI  # Used to store molecule ID, same as 10x default.
 num_mol_tag: MN # Used to store number of molecules per barcode
 sequence_tag: RX    # Used to store original barcode sequence in bam file. 'RX' is 10x genomic default
 genome_reference:  # Path to indexed reference
-read_mapper: bowtie2  # Choose bwa, bowtie2 or minimap2
+read_mapper: bowtie2  # Choose bwa or bowtie2 or minimap2
```

Comment:
Hm, why did you change the grammar here? This was quite standard English before.

Reply:
I think I made a mistake while rebasing, will change.